lib/wrappers/genesisProtocol.ts

```diff
@@ -67,20 +67,13 @@ export class GenesisProtocolWrapper extends IntVoteInterfaceWrapper implements S
   public async stake(options: StakeConfig =
     {} as StakeConfig & TxGeneratingFunctionOptions): Promise<ArcTransactionResult> {
 
-    const defaults = {
-      onBehalfOf: null,
```

Comment:
im actually not sure we should remove this from arc.js, if it exists in arc we should probably support it, even if im worried about the feature in general

Reply:
I felt that our concerns about it were significant enough to warrant erring on the cautious side.